src/test/scala/beam/agentsim/SingleModeSpec.scala

```diff
@@ -62,8 +63,24 @@ class SingleModeSpec
     beamConfig = BeamConfig(config)
 
     // Have to mock a lot of things to get the router going
+    //    val services: BeamServices =
```

Comment:
Can you remove these commented lines as it's just potentially confusing white noise.